### PR TITLE
EOS-13627 LogRotate Improvement: Rotate cortx-ha logs hourly basis in case it has exceeded size limit 

### PR DESCRIPTION
**Pull request by VijayThakkar132**
Signed-off-by: Vijay Thakkar <vijaykumar.thakkar@seagate.com>

## Problem Statement
<pre>
  <code>
  Story Ref (if any):
    EOS-13627 porting to main branch
  </code>
</pre>
## Unit testing on RPM done
<pre>
  <code>
  Yes
  </code>
</pre>
## Problem Description
<pre>
  <code>
    Rotate cortx-ha logs hourly basis in case it has exceeded size limit (porting to main branch)
  </code>
</pre>
## Solution
<pre>
  <code>
    Moved config file at /etc/logrotate_hourly.d/ dir and other improvement.
  </code>
</pre>
## Unit Test Cases
<pre>
  <code>
    Verified on VM and observed that rotation is happening as per the expectation.
  </code>
</pre>

### CODE REVIEW

**Review by ajay-paratmandali — APPROVED** [review]

File: conf/logrotate/cortx_ha_log.conf [hunk 1]
```diff
@@ -4,7 +4,9 @@
     rotate 10
     size 10M
     compress
+    dateext
```
Comment (ajaysrivas) [inline conf/logrotate/cortx_ha_log.conf:7]: https://github.com/Seagate/cortx-ha/pull/209 has 3 changed files while this one has two files ?

File: conf/script/ha_setup [hunk 2]
```diff
@@ -39,8 +39,11 @@ post_install() {
     mkdir -p /var/log/seagate/cortx/ha/
     chown -R csm /var/log/seagate/cortx/ha/
```
Comment (ajay-paratmandali) [inline conf/script/ha_setup:40]: https://github.com/Seagate/cortx-ha/tree/main/conf/etc/cron.hourly

Do we need this dir if not can we remove it

File: conf/logrotate/cortx_ha_log.conf (hunk 1 above)
Comment (VijayThakkar132) [reply conf/logrotate/cortx_ha_log.conf:7]: Earlier PR related to log rotate was not ported to main branch. And just now got to know that Ajay P has already merged those changes yesterday to main branch. Need to revert those changes also.

Comment (VijayThakkar132) [reply conf/logrotate/cortx_ha_log.conf:7]: Rebased with latest main and resolved merge conflicts

File: conf/script/ha_setup (hunk 2 above)
Comment (VijayThakkar132) [reply conf/script/ha_setup:40]: done

**Review by ajaysrivas — APPROVED** [review]

**Pull request merged**